### PR TITLE
Fix deprecation and add baseline

### PR DESCRIPTION
**Pull request by jderusse**
This PR fix few deprecations 

And also uses deprecation baseline to track and ignore deprecation we won't fix

### CODE REVIEW

File: src/Integration/Laravel/Queue/tests/baseline.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "location": "AsyncAws\\Illuminate\\Queue\\Tests\\Unit\\AsyncAwsSqsQueueTest::testPush",
+        "message": "Ramsey\\Uuid\\Uuid implements the Serializable interface, which is deprecated. Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary)",
```
Comment (jderusse) [inline src/Integration/Laravel/Queue/tests/baseline.json:4]: Hm... I wonder if we should really ignore this deprecation...
This is a true deprecation that should/will be fixed.  But this is an indirect deprecation (triggered by a 3rd party).

If we ignore this deprecation, then we'll better have to ignore all the indirect deprecation and only use `max[self]=0` instead of `max[total]=0`

TL;DR = should we care about indirect deprecations or just ignoring them?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -154,14 +154,21 @@ jobs:
       - name: Run tests
         env:
           SYMFONY_PHPUNIT_VERSION: ${{ matrix.phpunit_version }}
-          SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.strategy == 'lowest' && '9999' || '1' }}
+          OVERRIDE_SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.strategy == 'lowest' && 'max[total]=9999' || '' }}
         run: |
           ok=0
+          if [ -n "$OVERRIDE_SYMFONY_DEPRECATIONS_HELPER" ]; then
+            export SYMFONY_DEPRECATIONS_HELPER=$OVERRIDE_SYMFONY_DEPRECATIONS_HELPER
+          fi
```
Comment (jderusse) [inline .github/workflows/ci.yml:162]: This avoid defining the `SYMFONY_DEPRECATIONS_HELPER` env variable and let the value defined in `phpunit.xml.dist`

File: src/Integration/Laravel/Queue/tests/baseline.json (hunk 1 above)
Comment (Nyholm) [reply src/Integration/Laravel/Queue/tests/baseline.json]: I think we should just ignore all indirect deprecations.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Nyholm) [reply .github/workflows/ci.yml:162]: Oh. This is what I was missing before. Thank you!

**Review by Nyholm — APPROVED** [review]
